packages/backend-api/src/Controller/V1/Product/ProductController.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\BackendApiBundle\Controller\V1\Product;
+
+use FOS\RestBundle\Controller\AbstractFOSRestController;
+use FOS\RestBundle\Controller\Annotations\Delete;
+use FOS\RestBundle\Controller\Annotations\Get;
+use FOS\RestBundle\Controller\Annotations\Patch;
+use FOS\RestBundle\Controller\Annotations\Post;
+use FOS\RestBundle\Controller\Annotations\Put;
+use FOS\RestBundle\Controller\Annotations\QueryParam;
```

Comment:
I would personally use `FOS\RestBundle\Controller\Annotations` and annotate with `@Annotations\Get("...` But feel free to leave it like it is.

Reply:
ok